src/kmers.nim

```diff
@@ -278,6 +284,20 @@ proc haskmer*(target: pot_t; query: Bin): bool =
         return true
     return false
 
+## Counts the number of shared unique kmers
+## @param pot_t * - a pointer to a pot_t
+## @param pot_t * - a pointer to a pot_t
+## @return int - number of shared kmers
+#
+proc uniqueShared*(a,b: pot_t): int =
+ make_searchable(a)
+ make_searchable(b)
+ result = 0
+
+ for k in a.ht.keys():
+  if(haskmer(b, k)):
```

Comment:
If you run `make pretty`, the indentation will be normalized. (4 spaces was Ivan's preference.)

Reply:
OK, will don.